packages/select/docs/FilterMultiSelect.stories.tsx

```diff
@@ -600,3 +601,114 @@ export const Async: StoryFn<typeof FilterMultiSelect> = args => {
   )
 }
 Async.decorators = [withQueryProvider]
+
+export const WithSectionHeader: StoryFn<typeof FilterMultiSelect> = () => {
+  const [selectedKeys, setSelectedKeys] = useState<Selection>(
+    new Set(["id-fe"])
+  )
+
+  const handleSelectionChange = (keys: Selection): void => setSelectedKeys(keys)
+
+  return (
+    <>
+      <FilterMultiSelect
+        onSelectionChange={handleSelectionChange}
+        selectedKeys={selectedKeys}
+        items={mockItems}
+        label="Engineer"
+        trigger={(): JSX.Element => (
+          <FilterMultiSelect.TriggerButton
+            selectedOptionLabels={getSelectedOptionLabels(
+              selectedKeys,
+              mockItems
+            )}
+            label="Engineer"
+          />
+        )}
+      >
+        {(): JSX.Element => (
+          <>
+            <FilterMultiSelect.SearchInput />
+            <FilterMultiSelect.ListBox>
+              {({
+                selectedItems,
+                unselectedItems,
+                disabledItems,
+                hasNoItems,
+              }): JSX.Element => (
+                <>
+                  {hasNoItems && (
+                    <FilterMultiSelect.NoResults>
+                      No results found.
+                    </FilterMultiSelect.NoResults>
+                  )}
+                  <FilterMultiSelect.ListBoxSection
+                    items={selectedItems}
+                    sectionName="Selected items"
+                  >
+                    {(item): JSX.Element => (
+                      <FilterMultiSelect.Option key={item.key} item={item} />
+                    )}
+                  </FilterMultiSelect.ListBoxSection>
+
+                  {unselectedItems.length > 0 && selectedItems.length > 0 && (
+                    <FilterMultiSelect.SectionDivider />
+                  )}
+
+                  <FilterMultiSelect.ListBoxSection
+                    items={unselectedItems}
+                    sectionHeader={
+                      <InlineNotification
+                        type="cautionary"
+                        persistent
+                        noBottomMargin
+                        headingProps={{
+                          tag: "span",
+                          variant: "heading-5",
+                          children: "Confidentiality protection",
+                        }}
+                      >
+                        Results for these filters are hidden to protect
+                        identities of individuals and small grounds
```

Comment:
Little bit nit picky and it wasn't shown in Kayd's example for some reason, but I'm pretty sure they're going to use this on the list below of `disabledItems`, to make this story a bit more of a real life example

Reply:
Makes sense to me 👍 will update

Reply:
Only problem is that disabled items aren't navigate-able in this implementation so you can't key over them, which mean the header will not read 🤔 Part of the opinion of [react aria
](https://react-spectrum.adobe.com/react-aria/useListBox.html#disabled-items) I suspect
![Screenshot 2023-07-20 at 11 12 09 am](https://github.com/cultureamp/kaizen-design-system/assets/36558508/0cb58bc0-1007-4f99-9506-72e582e4bdc3)